packages/rocketchat-livechat/package.js

```diff
@@ -139,14 +141,18 @@ Package.onUse(function(api) {
 	api.addFiles('server/models/LivechatPageVisited.js', 'server');
 	api.addFiles('server/models/LivechatTrigger.js', 'server');
 	api.addFiles('server/models/indexes.js', 'server');
+	api.addFiles('server/models/LivechatInquiry.js', 'server');
 
 	// server lib
 	api.addFiles('server/lib/Livechat.js', 'server');
+	api.addFiles('server/lib/QueueMethods.js', 'server');
+	
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2726404244/issues/source?bid=3199634&fileBranchId=3348698#l149)